guide/docs/interactions/buttons.mdx

```diff
@@ -88,13 +56,203 @@ async def help_listener(inter: disnake.MessageInteraction):
+
+        # Now that the View's children have been disabled, we edit the message by sending the updated View.
+        # Since the View has timed out, it does not need to be explicitly stopped.
+        await self.message.edit(view=self)
+
+        # You may also remove all components by calling `self.clear_items()` which returns the View with
+        # no components.
+        # Sending this updated View in the `message.edit` will remove the components from the message.
+        # await self.message.edit(view=self.clear_items())
+
+    @disnake.ui.button(label="Yes", style=disnake.ButtonStyle.success)
+    async def yes(self, button: disnake.ui.Button, inter: disnake.MessageInteraction):
+
+        await inter.response.send_message("Contact us at https://discord.gg/disnake!")
+
+        # Update the View by disabling the buttons, then editing the original message
+        # the View was attached to, not the response message sent above.
+        # You could also do this as part of the interaction.response by using
+        # `inter.response.edit_message` which would handle the required response
+        # and editing of the original message.
+        self.disable_children()
+        await inter.message.edit(view=self)
+        # We still want to explicitly stop the View after a user has interacted
+        # to ensure it's no longer usable.
+        self.stop()
+
+    @disnake.ui.button(label="No", style=disnake.ButtonStyle.danger)
+    async def no(self, button: disnake.ui.Button, inter: disnake.MessageInteraction):
+
+        await inter.response.send_message("Got it. Signing off!")
 
-## Receiving button callback
+        # Update the View by disabling the buttons, then editing the original message
+        # the View was attached to, not the response message sent above.
+        # You could also do this as part of the interaction.response by using
+        # `inter.response.edit_message` which would handle the required response
+        # and editing of the original message.
+        self.disable_children()
+        await inter.message.edit(view=self)
+        # We still want to explicitly stop the View after a user has interacted
+        # to ensure it's no longer usable.
+        self.stop()
+
+
+...
+...
+# The slash command that will respond with a message and the View.
+@bot.slash_command()
+async def buttons(inter: disnake.ApplicationCommandInteraction):
+
+    # Here we need to construct the TimeoutButtonView so that it can be sent with
+    # the slash command response message. Since we do not have the message object yet,
+    # we assign it after the response has been sent.
+    view = TimeoutButtonView(timeout=120)
+    await inter.response.send_message("Need help?", view=view)
+
+    # Because this type of interaction response does not return a message, we will need
+    # to fetch the message and assign it to `View.message` to be used if `on_timeout` is called.
+    View.message = await inter.original_response()
```

Comment:
```suggestion
    # Because interaction responses do not return a message, we will need
    # to fetch the message and assign it to `View.message` to be used if `on_timeout` is called.
    view.message = await inter.original_response()
```